st3/lsp_utils/npm_client_handler.py

```diff
@@ -61,6 +62,10 @@ def get_additional_variables(cls) -> Dict[str, str]:
         })
         return variables
 
+    @classmethod
+    def get_additional_paths(cls) -> List[str]:
+        return [path.dirname(cls._node_bin())]
```

Comment:
We shouldn't need to return anything here when using node on PATH but it should hurt to include it either.

Reply:
Even in the subprocess, I was curious to find out if two same named commands exist in the path, which one will win... Answer: Apparently the first one that is matched:

https://superuser.com/questions/506450/if-two-separate-path-directories-contain-a-same-named-executable-how-does-windo